.github/workflows/ci.yml

```diff
@@ -72,27 +67,70 @@ jobs:
         uses: actions/checkout@v1
       - name: Install system packages
         run: |
-          apt-get update
-          apt-get install -y --no-install-recommends \
-            unzip
+          apk add \
+            unzip \
       - name: Install mongodb PHP extension
         if: matrix.php != '7.1'
         run: |
+          apk add $PHPIZE_DEPS
           pecl install mongodb-1.5.5
           docker-php-ext-enable mongodb
       - name: Disable PHP memory limit
         run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
       - name: Install Composer
-        run: curl -sS -o - https://raw.githubusercontent.com/composer/getcomposer.org/ba13e3fc70f1c66250d1ea7ea4911d593aa1dba5/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
+        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/fb22b78362d31c0d2bf516d1f8cdfd2745caa431/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
```

Comment:
can't we use a variable for this composer sha? may ease maintenance?

Reply:
GitHub workflow files don't support YAML anchors yet... :disappointed: